src/core/main.cpp

```diff
@@ -510,6 +510,15 @@ int main( int argc, char * * argv )
 		Engine::getSong()->loadProject( file_to_load );
 		printf( "done\n" );
 
+		if ( Engine::getSong()-> hasErrors() )
+		{
+			const char * summary = Engine::getSong()->errorSummary()->toStdString().c_str();
+
+			fputs( "\n", stderr );
+			fputs( summary, stderr );
+			fputs( "\n", stderr );
```

Comment:
Why are you converting these to c strings? Keep it as a QString and use a Qt function to display the string.